app/bundles/DashboardBundle/Assets/js/dashboard.js

```diff
@@ -1,7 +1,82 @@
 //DashboardBundle
+Mautic.widgetUrl = '/s/dashboard/widget/';
```

Comment:
It could be absolute URL
On on mautic in subdirectory domaint.tld/mautic/ it return 404 

![image](https://user-images.githubusercontent.com/462477/99801700-c0ee4080-2b36-11eb-9935-05cd4578e4d8.png)

Reply:
fixed in https://github.com/mautic/mautic/pull/9386/commits/1f75c31a876f77cb135229950a5870678c82087c